pandas/tests/io/json/test_pandas.py

```diff
@@ -1601,3 +1601,16 @@ def test_json_indent_all_orients(self, orient, expected):
     def test_json_negative_indent_raises(self):
         with pytest.raises(ValueError, match="must be a nonnegative integer"):
             pd.DataFrame().to_json(indent=-1)
+
+    @pytest.mark.parametrize(
```

Comment:
Since you only have one pair of values shouldn't need to parametrize at all; just add to body of test

Reply:
True, I initially thought of going with multiple test cases. Should I rewrite it?

Reply:
Removed parametrize